extraction/uint63.mli

```diff
@@ -0,0 +1,97 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
```

Comment:
Hope nobody polices this library to become LGPL.

Reply:
Ugh. We do legally have to relicense coq-simple-io (I thought not at first). Or we quarantine this in another library, but that seems too much trouble.